### PR TITLE
add repo url

### PR DESCRIPTION
**Pull request by zeke**
...so it shows up on the npm package page

### CODE REVIEW

**Comment by camwiegert** [comment]
@zeke Good call. Thanks!

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "in-view",
   "version": "0.4.2",
   "description": "Get notified when a DOM element enters or exits the viewport.",
+  "repository": "https://github.com/camwiegert/in-view",
```
Comment (camwiegert) [inline package.json]: Don't we want this in a slight different format? Per [the docs](https://docs.npmjs.com/files/package.json#repository):

```json
"repository": {
    "type": "git",
    "url": "https://github.com/camwiegert/in-view.git"
}
```

Comment (camwiegert) [inline package.json]: Looks like we might also be able to...

```json
"repository": "github:camwiegert/in-view"
```

**Comment by zeke** [comment]
The npm publish process will convert it to that more complex object behind the scenes. I'm partial to URLs because they are the [Uniform Way to Locate Resources](http://adam.herokuapp.com/past/2010/3/30/urls_are_the_uniform_way_to_locate_resources/)

**Comment by camwiegert** [comment]
Works for me. Thanks again.

**Pull request merged**